style.css

```diff
@@ -109,3 +118,236 @@ ul {
 .social-icons li {
   padding-right: 18px;
 }
+
+/* Project Section */
+.project-section {
+  display: flex;
```

Comment:
[OPTIONAL]
- [ ] Good job using `display: flex;` in the main container for the project cards, however the project requirement is to use a `display: grid` for the main container of the work section. ↔️ 👍🏿